content/building/environment-variables.md

```diff
@@ -26,6 +26,7 @@ Here you'll find some of the read-only environment variables explained.
 
 | **Environment variable** | **Value**                                                                                                                                                       |
 | ------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ANDROID_SDK_ROOT         | Absolute path to Android SDK and tools                                                                                                                          |
```

Comment:
Please don't add it here. This table is for `Codemagic read-only environment variables`. We have a bunch of other unrelated to codemagic variables like 
```
LC_CTYPE, LC_ALL, LANG, LANGUAGE, TMPDIR, USER, HOME, RBENV_VERSION, JAVA_HOME, ANDROID_HOME, ANDROID_PREFS_ROOT, NDK_HOME, ANDROID_NDK_HOME, GRADLE_HOME, ADB_INSTALL_TIMEOUT, FLUTTER_ROOT, GIT_MERGE_AUTOEDIT, SNAPCRAFT_BUILD_ENVIRONMENT
```

and `ANDROID_SDK_ROOT` is just one among them. It would be proper to add it in a separate page or separate table, not here

Reply:
I don't agree, it should be one place for all exported vars. and it's fine to no mention system and rare used vars

Reply:
Great comment @BondarenkoStas. I was thinking about the same thing but didn't know the other variables.

I'll add a section for the new table and ping you for a review of the descriptions.

Reply:
Created new PR for this https://github.com/codemagic-ci-cd/codemagic-docs/pull/330